README.md

```diff
@@ -1,44 +1,76 @@
+    <td><img src="./home_screen.png"></td>
+    <td><img src="./details_screen.png"></td>
+  
+  </tr>
+ </table>
 
-The page will reload if you make edits.<br />
-You will also see any lint errors in the console.
+A simple web app to show trending photos from [Pixabay](https://pixabay.com/api/docs/), based on likes.
+Original design idea by[ Nelson Sakwa on Behance](https://www.behance.net/sakwadesignstudio).
 
-### `npm test`
+## Built With
 
-Launches the test runner in the interactive watch mode.<br />
-See the section about [running tests](https://facebook.github.io/create-react-app/docs/running-tests) for more information.
+- React
+- Redux
 
-### `npm run build`
+## Live Demo
 
-Builds the app for production to the `build` folder.<br />
-It correctly bundles React in production mode and optimizes the build for the best performance.
+[Live Demo Link](https://bavon101-pixel.herokuapp.com/)
 
-The build is minified and the filenames include the hashes.<br />
-Your app is ready to be deployed!
+## Getting Started
 
-See the section about [deployment](https://facebook.github.io/create-react-app/docs/deployment) for more information.
+To get a local copy up and running follow these simple example steps.
 
-### `npm run eject`
+### Prerequisites
 
-**Note: this is a one-way operation. Once you `eject`, you can’t go back!**
+A Web Browser (preferably Google Chrome)
 
-If you aren’t satisfied with the build tool and configuration choices, you can `eject` at any time. This command will remove the single build dependency from your project.
+### Setup
 
-Instead, it will copy all the configuration files and the transitive dependencies (Webpack, Babel, ESLint, etc) right into your project so you have full control over them. All of the commands except `eject` will still work, but they will point to the copied scripts so you can tweak them. At this point you’re on your own.
+- Clone the GitHub Repository
+- Go to the Project Directory
+- Run ```npm install```
+- Run ```npm run build``` to build the project
 
-You don’t have to ever use `eject`. The curated feature set is suitable for small and middle deployments, and you shouldn’t feel obligated to use this feature. However we understand that this tool wouldn’t be useful if you couldn’t customize it when you are ready for it.
+### Usage
 
-## Learn More
+- Run ```npm run start``` to start the live server
 
-You can learn more in the [Create React App documentation](https://facebook.github.io/create-react-app/docs/getting-started).
+## Authors
 
-To learn React, check out the [React documentation](https://reactjs.org/).
+👤 **Author**
+
+- GitHub: [@Bavon101](https://github.com/Bavon101)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/akumu-bavon-335416193/)
+
+
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+## Acknowledgments
+
+- Microverse
+
```

Comment:
- [ ] In keeping with the Creative Commons License requirement, kindly add an acknowledgment for Nelson Sakwa, the author of the original design.